.github/workflows/primer_comment.yaml

```diff
@@ -0,0 +1,140 @@
+# Most of this is inspired by the mypy primer
+# See: https://github.com/hauntsaninja/mypy_primer
+# This is the primer job that creates the comment on the PR
+# It needs to trigger on workflow_run instead of pull_request
+# as we need repository wide access to create a comment
+
+name: Primer / Comment
+
+on:
+  workflow_run:
+    workflows: [Primer / Run]
+    types:
+      - completed
+
+env:
+  CACHE_VERSION: 1
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  primer-comment:
+    name: Run
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          version: 12
+      - run: npm install @octokit/rest
+      - name: Check out code from GitHub
+        uses: actions/checkout@v3.0.2
+      - name: Set up Python 3.8
+        id: python
+        uses: actions/setup-python@v3.1.2
+        with:
+          python-version: 3.8
+
+      # Restore cached Python environment
+      - name: Generate partial Python venv restore key
+        id: generate-python-key
+        run: >-
+          echo "::set-output name=key::venv-${{ env.CACHE_VERSION }}-${{
+            hashFiles('setup.cfg', 'requirements_test.txt', 'requirements_test_min.txt')
+          }}"
+      - name: Restore Python virtual environment
+        id: cache-venv
+        uses: actions/cache@v3.0.2
+        with:
+          path: venv
+          key: >-
+            ${{ runner.os }}-${{ steps.python.outputs.python-version }}-${{
+            steps.generate-python-key.outputs.key }}
+          restore-keys: |
+            ${{ runner.os }}-${{ steps.python.outputs.python-version }}-venv-${{ env.CACHE_VERSION }}-
+      - name: Create Python virtual environment
+        if: steps.cache-venv.outputs.cache-hit != 'true'
+        run: |
+          python -m venv venv
+          . venv/bin/activate
+          python -m pip install -U pip setuptools wheel
+          pip install -U -r requirements_test.txt
```

Comment:
This applies to a different workflow file, but just commenting here. 

Also something I should have mentioned two PRs ago: I wonder if we want to use bleeding-edge `astroid` for these linting runs. If there are pending changes in astroid that cause false positives/negatives/crashes with someone's pylint PR, the pylint PR is our chance to address them before the contributor loses interest.

Reply:
As to `astroid` I'm not actually sure. There are also cases where bleeding edge astroid is incompatible with latest pylint. So both can cause issues.

Perhaps we should default to one case and then turn on the other when we need to? We could just add a flag to `run` mode. Not sure if that would be a blocker though. Perhaps we should just let this run as a non required workflow? There are many edge cases that we probably haven't considered yet.

_I also thought about only running this workflow when a PR has a certain label. That way we can control this a little better while the tool is still in it's beta phase._

As for cyclic import. I have no idea actually. It couldn't get it to reproduce locally easily and thought about all the problems we have with the message so just thought I would disable it.

Reply:
> As for cyclic import. I have no idea actually. It couldn't get it to reproduce locally easily and thought about all the problems we have with the message so just thought I would disable it.

I got it just now locally on the first try, so maybe I can try to help debug. Looking at the JSON output of the runs I'm seeing a lot fatal errors ("no module named src/astroid") etc., so I'm wondering if we're not invoking the runs correctly.

Reply:
> There are also cases where bleeding edge astroid is incompatible with latest pylint. So both can cause issues.

That could slow down astroid contributors dealing with pre-commit, but would that affect pylint contributors? It's true we don't have to block, so if you prefer I'll open a PR and we can discuss there. I think we should do it. If pending `astroid` changes broke `pylint` main, we want to know immediately and block all pylint dev anyway, IMO.

Reply:
> I also thought about only running this workflow when a PR has a certain label. That way we can control this a little better while the tool is still in it's beta phase.

Neat! So for now, any functional change or nontrivial refactor gets "primer-diff-messages"?

Reply:
Yeah that would be a possibility. At some point I think we want to turn this on based on paths so we don't need to keep adding the label, but it would be a possibility to start with "label-only".

Do you want me to change it do work like that?

Reply:
Let's not do it now. Maybe the paths can be done eventually and we won't have to spend time on labels.